app_dev.yaml

```diff
@@ -95,8 +82,35 @@ handlers:
   upload: extensions/(interactions|rich_text_components)/(.*)/directives/(.*\.(html))
   secure: always
   expiration: "0"
+- url: /extensions/visualizations/(.*\.html)
+  static_files: extensions/visualizations/\1
+  upload: extensions/visualizations/(.*\.html)
+  secure: always
+  expiration: "0"
+- url: /extensions/objects/templates/(.*\.html)
+  static_files: extensions/objects/templates/\1
+  upload: extensions/objects/templates/(.*\.html)
+  secure: always
+  expiration: "0"
+# Please do not modify the below comment.
+# It is required for correct generation of app.yaml in prod_env.
+# DEVELOPMENT STATIC END
+
+# COMMON FOR DEV AND PROD
+- url: /third_party/generated
+  static_dir: third_party/generated
+  secure: always
+  expiration: "0"
+# Serve js scripts and css files under core/templates/dev/head.
+# This regex allows us to recursively serve js scripts.
+# "\1" inserts text captured by the capture group in the URL pattern.
+- url: /templates/dev/head/(.*\.(css))$
+  static_files: core/templates/dev/head/\1
+  upload: core/templates/dev/head/(.*\.(css))$
+  secure: always
+  expiration: "0"
 # Serve js scripts for gadgets, interactions, rich_text_components, custom ckeditor plugins
```

Comment:
Drop "gadgets" from this docstring, we don't have those any more.

Reply:
Done.